src/components/settings/PreferenceSettings.tsx

```diff
@@ -208,9 +208,9 @@ class PreferenceSettings extends Component<
               value={this.state.settings.language}
               onChange={this.onChange("language", "value")}
             >
-              {languages.map((language: string) => (
-                <option key={language} value={language}>
-                  {language}
+              {languages.map((language: any) => (
```

Comment:
Remove the type annotation here, as TypeScript will automatically infer the correct type.
```suggestion
              {languages.map(language => (
```